### ISSUE TITLE
Feature Request: "Associated Publication" field 

### ISSUE THREAD

**Issue by dvirlar2** [NCEAS/metacatui#2051]
**Describe the feature you'd like:**
I think PIs edit published datasets (those with DOIs) to include citations for recently published papers because (1) they don't recognize the utility of the Citation button ([related issue here](https://github.com/NCEAS/metacatui/issues/2050)), and because (2) putting a paper's citation in the abstract is a nice easy way to see that a paper from the dataset author's has been published.

If possible, I think either a field for "Associated Publication" in the early contents of the dataset would be useful in putting that information front and center. In a perfect world, it'd pull from a cited paper in the citations that is maybe declared the "associated publication" by the PIs. 

Basically creating some way of directly showing a main paper from the PIs has been published, but in a way that doesn't create a new version of the dataset (for the same reasons as mentioned in [issue above](https://github.com/NCEAS/metacatui/issues/2050)).

**Comment by robyngit** [comment]
In discussion with @dvirlar2, it's clear that PIs have a lot of interest in the citation feature, and would like to make the list of citations more prominent in the display of the dataset. Here are a few ideas that we discussed:
- Move the citation list from the button/modal window into the main view of the dataset, near the abstract. (This would also help with making the Add Citation button easier to find, https://github.com/NCEAS/metacatui/issues/2050)
- Datasets are frequently most closely associated with a _single_ publication (the "associated publication"). For example, data may be collected for that publication, and more details about data collection methods can be found there. It would be nice to have some way to make this very closely associated publication very prominent in the metadata display. PIs currently tend to update the abstract for this purpose.

**Comment by dvirlar2** [comment]
Another PI recently asked for publications to be added to the abstract field of their datasets (Feb 14 messages, [ticket 25257](https://support.nceas.ucsb.edu/rt/Ticket/Display.html?id=25257)), so I just wanted to give this a gentle boost 😇

**Comment by robyngit** [comment]
Thanks @dvirlar2, this is helpful! :)
